### ISSUE TITLE
White page after "Next" on creating new poll

### ISSUE THREAD

**Issue by RealRancor** [v1r0x/polls#46]
Using Polls 0.7.0 on oC 8.2.3 hitting "Next" on the "Create new poll" page leads to a white page.

Steps to reproduce:

1. Create new poll
2. Enter simple title and description
3. Leave the defaults "Registered users only",  "Event schedule" and no "Expires"
4. Choose one single day and one single hour
5. Turn the single cell to green
6. Hit "Next"

After doing this steps the /index.php/apps/polls/create page redirects to an empty/white /index.php/apps/polls/. Hitting Enter in the URL bar (not reloading the page with F5) refreshes the page and shows the newly created poll.

There is absolutely nothing logged in the data/owncloud.log, php error log or nginx error log so this is not an internal server error.

The only interesting part could be:

```
xxx - - [19/Mar/2016:00:25:07 +0100] "POST /index.php/apps/polls/insert HTTP/1.1" 307 0 "-" "Mozilla/5.0"
xxx - - [19/Mar/2016:00:25:07 +0100] "POST /index.php/apps/polls/ HTTP/1.1" 405 0 "-" "Mozilla/5.0"

```

where it seems that the POST for inserting the poll is redirected to a POST on /index.php/apps/polls/ which is failing with a 405/Method Not Allowed.

The same is also happening when e.g. deleting a poll.

**Comment by v1r0x** [comment]
This is a bug in oc core and has been fixed for 9.0. For older versions you have to edit this line: https://github.com/owncloud/core/blob/v8.2.3/lib/public/appframework/http/redirectresponse.php#L47 and replace `Http::STATUS_TEMPORARY_REDIRECT` with `Http::STATUS_SEE_OTHER`.

**Comment by RealRancor** [comment]
Grmlll, nevermind. Thought this is not related to https://github.com/owncloud/core/pull/22497 as thought thats already included in 8.2.3. But i was wrong. :-/
